src/components/Panel/Panel.tsx

```diff
@@ -36,7 +36,7 @@ const Panel = (props: Props) => {
   return (
     <Card border={color}>
       {title && (
-        <Card.Header style={{ textAlign: 'left' }}>
+        <Card.Header onClick={() => setOpen(!open)} style={{ textAlign: 'left' }}>
```

Comment:
I noticed if the panel is not collapsible, then the onclick event still fires causing the panel to collapse. 

If the panel is not collapsible, then it should should not collapse when clicking the header.

Reply:
Added a complementary test and a conditional to ensure the panel is collapsible before toggling the collapse.
I am still new to testing and looking to practice. I'm not too sure exactly too what extent we should test these components. Should I also create a test to make sure clicking on the toggle icon collapses and opens the panel? I hadn't seen one present so I wasn't sure but if there are any missing tests or mistakes send me a reply!

Reply:
In general, if it can be tested, we should probably test it. 

it sounds like a test to check that clicking on the icon collapses and opens the panel seems like a good test!